test/app/views/layouts/application.html.erb

```diff
@@ -6,5 +6,6 @@
 </head>
 <body>
   <%= yield %>
+  <%= content_for?(:view_component_preview_source) ? yield(:view_component_preview_source) : yield %>
```

Comment:
why do we render `view_component_preview_source` in the application template?

Reply:
Also isn't this code double yielding if there is no `view_component_preview_source`

Reply:
Fixed the double yielding in 6b9fccd62f888cca8bbcf3889e1f56734897cd37. 

> why do we render view_component_preview_source in the application template?
By default, previews are rendered in the application layout. We need to render `view_component_preview_source ` in order to append the example source code after the preview. `content_for?(:view_component_preview_source)` only returns true if we render a preview.